### PR TITLE
Added Forgot password Feature

### PR DESCRIPTION
**Pull request by NancyAanchal**
Added forgot password feature in Login ad it works

### CODE REVIEW

**Comment by NancyAanchal** [comment]
I figured Supabase doesn't have a Built In 'email not registered error message. Hence we get this message even if the email is not registered . I tried a bunch of logics in Auth context but it didn't seem to work. I didn't know what to do so I pushed it. What are your opinions ?
image

File: nepalingo-web/src/pages/Login.tsx [hunk 1]
```diff
@@ -80,6 +80,10 @@ const Login: React.FC = () => {
     setPassword("");
   };
 
+  const PasswordReset = () => {
```
Comment (SansCaar) [inline nepalingo-web/src/pages/Login.tsx:83]: as the function is just navigationg it would be convinient to  just be called in the onClick

**Review by SansCaar — COMMENTED** [review]
Yeah the code looks fine

**Comment by SansCaar** [comment]
The link in the email redirects to local host could you please redirect it to our domain

**Comment by SansCaar** [comment]
We can change the ui in another ticket

**Review by SansCaar — DISMISSED** [review]

**Review by SansCaar — APPROVED** [review]

**Pull request merged**